history-of-the-quakers-v2/original/08-book-3-chapter-14.adoc

```diff
@@ -775,7 +835,10 @@ Particularly in the year 1731 this weighty subject
 seems to have engaged their very serious consideration,
 which produced the following lively recommendation.
 
-"`Inasmuch as we have a large body of youth growing up, the offspring of friends,
+[.embedded-content-document]
```

Comment:
Should this be a minute? It didn't exactly say it, but it seems in line with the previous minutes, but I wasn't sure.

Reply:
yeah, i'm not sure either. i think i'm comfortable leaving it without a type like you did. 👍